server/util-server.js

```diff
@@ -449,3 +451,33 @@ exports.errorLog = (error, outputToConsole = true) => {
         }
     } catch (_) { }
 };
+
+/**
+ * Returns a color code in hex format based on a given percentage:
+ * 0% => hue = 10 => red
+ * 100% => hue = 90 => green
+ *
+ * @param {number} percentage, float, 0 to 1
```

Comment:
```suggestion
 * @param {number} percentage float, 0 to 1
```

Perhaps remove the comma after each param